apps/codeforafrica/src/payload/collections/Posts.js

```diff
@@ -39,7 +40,9 @@ const Posts = {
       },
     }),
     content({ minRows: 1, required: true, localized: true }),
-    slug(),
+    slug({
+      index: true,
```

Comment:
Add this to `payload/fields/slug.js`. Should be beneficial to everything that's using a `slug`.

Reply:
Turns out this is already indexed

Reply:
Turns out this is already indexed